lib/linux_admin/network_interface.rb

```diff
@@ -148,7 +148,7 @@ def start
     #
     # @return [Boolean] whether the command succeeded or not
     def stop
-      Common.run(Common.cmd("ifdown"), :params => [@interface]).success?
+      Common.run(Common.cmd("ifconfig"), :params => [@interface, "down"]).success?
```

Comment:
From the `ifconfig` man-page:

```plain
NOTE
       This program is obsolete!  For replacement check ip addr and ip link.  For statistics use ip -s link.
```

Can we find the newer version of this process rather than using an obsolete tool?